assets/konnectivity/konnectivity-server-services.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: konnectivity-server
+  labels:
+    app: kube-apiserver
+spec:
+  type: NodePort
+  selector:
+    app: kube-apiserver
+  ports:
+    - port: {{ .KonnectivityAgentPort }}
+      targetPort: {{ .KonnectivityAgentPort }}
+{{- if .KonnectivityServerAgentNodePort }}
```

Comment:
Why is this called `ServerAgent` when it is for the `Server` not the `Agent`?

Reply:
Hard to tell what is the right variable name here. As I wrote about, this service port is for the Agents.
Maybe the `KonnectivityAgentNodePort` is a better name. We can change this. Thanks!

Reply:
Discussed with Tomi, I will rename this to `KonnectivityAgentNodePort`. Thanks!